src/signalhandler.cc

```diff
@@ -363,11 +392,15 @@ void InstallFailureSignalHandler() {
   for (size_t i = 0; i < ARRAYSIZE(kFailureSignals); ++i) {
     CHECK_ERR(sigaction(kFailureSignals[i].number, &sig_action, NULL));
   }
+#elif defined(OS_WINDOWS)
+  typedef void (*SignalHandlerPointer)(int); 
+  SignalHandlerPointer previousHandler;  
+  previousHandler = signal(SIGABRT, &FailureSignalHandler); 
```

Comment:
Oh [here it is](https://github.com/google/glog/pull/168/files#diff-8b7c2e002f1814c322f81e0c0ecb3e7cR1481).

The signal handler was being on the thread on which it was installed, not a dedicated SEH thread (I was mis-remembering), and importantly, not the thread on which the signal was raised.